src/libData/AccountData/AccountStoreSC.h

```diff
@@ -37,7 +37,7 @@ class ScillaBCInfoProvider : public ScillaBCInfo {
         m_rootHash(rootHash),
         m_scillaVersion(scillaVersion) {}
 
-  uint64_t getCurBlockNum() const { return m_curBlockNum; }
+  const uint64_t& getCurBlockNum() const { return m_curBlockNum; }
   dev::h256 getRootHash() const { return m_rootHash; }
```

Comment:
Sorry May be I wasn’t clear. Can we change it for all rest of 3 getter functions?